java/clients/src/main/java/sleeper/clients/deploy/DeployNewInstance.java

```diff
@@ -125,6 +142,9 @@ public void deploy() throws IOException, InterruptedException {
                 Files.readString(templatesDirectory.resolve("schema.template")),
                 loadProperties(templatesDirectory.resolve("tableproperties.template")),
                 tableName);
+        if (splitPointsFile != null) {
+            tableProperties.set(SPLIT_POINTS_FILE, splitPointsFile.toString());
+        }
```

Comment:
There's a null check in SleeperProperties.set. It might be clearer to use `Objects.toString(splitPointsFile, null)`.